cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -3582,10 +3606,25 @@ class RaysView extends PolyShapeView {
     _makeEditable() {
         PolyShapeView.prototype._makeEditable.call(this);
         this._drawVanishingPoint();
+        this._lines.forEach((lineElement, index) => {
+            lineElement.on('dblclick', () => {
+                this._controller.model().removePoint(index * 2);
+            });
+            lineElement.remember('_selectHandler').pointSelection.set.members.forEach((point) => {
+                point = $(point.node);
+                point.on('dblclick', () => {
+                    this._controller.model().removePoint(index * 2);
+                    // e.stopPropagation();
```

Comment:
why commented out?

Reply:
I experimented with the code when it didn't work. Removed this line.